pyxem/utils/io_utils.py

```diff
@@ -135,30 +137,621 @@ def load_hspy(filename, lazy=False, assign_to=None):
     return s
 
 
-def load_mib(filename, scan_size, sum_length=10):  # pragma: no cover
-    """Load a medipix hdr/mib file.
 
+def load_mib(mib_filename, reshape=True):
+    """Read a .mib file using dask and return as a lazy pyXem / hyperspy signal.
+    
     Parameters
     ----------
-    filename : string
-        File path and name to .hdr file.
-    scan_size : int
-        Scan size in pixels, allows the function to reshape the array into
-        the right shape.
-    sum_length : int
-        Number of lines to sum over to determine scan fly back location.
+    mib_filename : str
+        The name of the .mib file to be read.
+    
```

Comment:
Add documentation for the "reshape" keyword argument here.

Reply:
This is now added:
`"reshape : boolean
        keywork argument to control reshaping of the stack (default is True).
        It attepmts to reshape using the flyback pixel."`